components/drops/calendar/DropPreviewModal.vue

```diff
@@ -21,11 +21,24 @@
             @click="emit('close')" />
         </header>
 
-        <div v-if="dropCalendar.dropStartTime" class="!mt-6">
-          <NeoButton no-shadow rounded @click="isCreateEventModalActive = true">
+        <div class="!mt-6 flex flex-wrap gap-6">
+          <NeoButton
+            v-if="dropCalendar.dropStartTime"
+            no-shadow
+            rounded
+            @click="isCreateEventModalActive = true">
             {{ $t('scheduled') }}<span class="text-neutral-5 mx-2">•</span
             >{{ formattedDate }}
           </NeoButton>
+
+          <NeoButton
+            v-if="dropCalendar.alias"
+            variant="secondary-rounded"
+            icon-left="sparkles"
+            :tag="NuxtLink"
+            :to="`/${urlPrefix}/drops/${dropCalendar.alias}`">
```

Comment:
In case we add `chain` to the calendar.
```suggestion
            :to="`/${dropCalendar.chain || urlPrefix}/drops/${dropCalendar.alias}`">
```

Reply:
updated [161c830](https://github.com/kodadot/nft-gallery/pull/10086/commits/161c830694fe2ea4b0e349e6ce6fee227d19aa0c)